conf/resinhup.conf

```diff
@@ -25,6 +25,8 @@ root_whitelist:
     etc/wpa_supplicant/entropy.bin
     .rce/key.json
     usr/bin/run-resinhup.sh
+    usr/bin/update-resin-supervisor
```

Comment:
Isn't /usr/bin/update-resin-supervisor already present in the rootfs? Why is it scp'ed by resinhup ssh wrapper script?